torch/_dynamo/variables/tensor.py

```diff
@@ -337,6 +337,24 @@ def call_method(
                 ),
                 **options,
             )
+        elif (
+            name == "add_" and len(args) == 1 and len(kwargs) == 1 and "alpha" in kwargs
```

Comment:
It's because `alpha` is a scalar (basically the same problem as with addcdiv). During tracing it calls `local_scalar_dense` too. How would the primtorch decomp work?